fastapi_jsonapi/schema_builder.py

```diff
@@ -488,7 +490,7 @@ def create_jsonapi_object_schemas(
         if use_schema_cache and schema in self.object_schemas_cache and includes is not_passed:
             return self.object_schemas_cache[schema]
 
-        schema.update_forward_refs(**registry.schemas)
+        schema.model_rebuild(_types_namespace=registry.schemas)
```

Comment:
I'am hesitant about this change, because now we get the `pydantic.errors.PydanticUndefinedAnnotation: name 'PostSchema' is not defined` which most likely means that `model_rebuild` works not as it should be or declared in the wrong place